openhands/sdk/tool/tool.py

```diff
@@ -292,7 +292,9 @@ def to_openai_tool(
         """
         action_type = action_type or self.action_type
 
-        action_type_with_risk = _create_action_type_with_risk(self.action_type)
+        action_type_with_risk = _create_action_type_with_risk(
+            action_type  # type: ignore[arg-type]
```

Comment:
I think we should avoid having type ignores they will be a source of pain in the future